src/core/thread/network_data_publisher.hpp

```diff
@@ -308,7 +308,7 @@ class Publisher : public InstanceLocator, private NonCopyable
         static constexpr uint32_t kExtraDelayToRemovePeferred =
             OPENTHREAD_CONFIG_NETDATA_PUBLISHER_EXTRA_DELAY_TIME_TO_REMOVE_PREFERRED;
 
-        static constexpr uint16_t kInfoStringSize = 50;
+        static constexpr uint16_t kInfoStringSize = 56;
```

Comment:
Thanks. I noticed the same looking at the logs from https://github.com/openthread/openthread/pull/8395, and was planning to increase it in this PR 😃.

Let's use `60` for this so we have more room (for possibly new `State` enum which may be longer).
```suggestion
        static constexpr uint16_t kInfoStringSize = 60;
```